makefile

```diff
@@ -1,14 +1,11 @@
 unittest-args = -m unittest discover --start-directory tests --top-level-directory .
 
-define doctest-cmd =
-cd docs && $(MAKE) doctest
-endef
-
```

Comment:
This does not need to exist. I've moved the contents of this declaration into the `docs-doctest` target.